packages/api/src/webhooks/cannon.ts

```diff
@@ -75,7 +82,8 @@ export default class WebhookCannon {
     if (newInterval > MAX_BACKOFF) {
       return lastInterval;
     }
-    return newInterval;
+    // RabbitMQ expects integer
+    return newInterval | 0;
```

Comment:
A little more expressive:
```suggestion
    return Math.floor(newInterval);
```

Reply:
But it is different - I like `| 0` more because it always returns integer. `Math.floor(1/0)` returns `Infinity`